mix.exs

```diff
@@ -28,6 +28,9 @@ defmodule ExDoc.Mixfile do
   defp deps do
     [
       {:earmark, "~> 1.1"},
+      # Syntax highlighting (independent of the markdown implementation)
+      {:makeup, "~> 0.5.1"},
+      {:makeup_elixir, "~> 0.6.0"},
```

Comment:
Shouldn't depending on makeup_elixir automatically bring makeup?

Reply:
Yeah, it should. We can depend only on makeup and makeup_elixir will be brought as a transitive dependency.

Reply:
You probably mean the other way around. :)